lock_free_timer.go

```diff
@@ -1,12 +1,9 @@
 package lft
 
 import (
-	"sync"
-	"sync/atomic"
 	"time"
 
 	metrics "github.com/rcrowley/go-metrics"
-	rand "github.com/remerge/go-xorshift"
 )
 
 func GetOrRegisterLockFreeTimer(name string, r metrics.Registry) metrics.Timer {
```

Comment:
to make it clear: `NewLockFreeTimer()` hasn't been changed intentionally to use a sample with buckets. The reason: the timer doesn't deal with buckets at all, so I don't know a reason to use it here; at least for now.